Description: Enable dbus in thunderbird to allow enigmail.
 firejail (0.9.58-1parrot4) testing; urgency=medium
 .
   * Fix thunderbird sandbox profile.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2019-02-12

--- firejail-0.9.58.orig/etc/thunderbird.profile
+++ firejail-0.9.58/etc/thunderbird.profile
@@ -43,5 +43,5 @@ writable-run-user
 # allow browsers
 # Redirect
 # Uncomment if you use enigmail
-# ignore nodbus
+ignore nodbus
 include firefox.profile
